### PR TITLE
chore: Fix and combine copyright linters

### PR DESCRIPTION
**Pull request by mayank99**
two commits:
- [`037a0fe`](https://github.com/iTwin/iTwinUI/commit/037a0fe576c9fec5ce079a45e8500117296e1865) moved a combined version of copyrightLinter to `scripts`, deleted old versions from `shared` and `configs`
- [`82bd6b`](https://github.com/iTwin/iTwinUI/commit/82bd6b12fcc82f6ed278857bff89dae7d896b014) ran `yarn lint:copyright` which added missing copyright headers to some `.js` files

### CODE REVIEW

**Review by FlyersPh9 — APPROVED** [review]

File: scripts/copyrightLinter.js [hunk 1]
```diff
@@ -22,10 +36,10 @@ const copyrightLine2 = `See LICENSE.md in the project root for license terms and
...
 const copyrightBannerHtml = `<!--\n  ${copyrightLine1}\n  ${copyrightLine2}\n-->`;
 const copyrightBannerJs =
-  '/*---------------------------------------------------------------------------------------------\n ' +
-  `* ${copyrightLine1}\n ` +
-  `* ${copyrightLine2}\n ` +
-  '*--------------------------------------------------------------------------------------------*/';
+  '/*---------------------------------------------------------------------------------------------\n' +
+  ` * ${copyrightLine1}\n` +
+  ` * ${copyrightLine2}\n` +
+  ' *--------------------------------------------------------------------------------------------*/';
```
Comment (rohan-kadkol) [inline scripts/copyrightLinter.js:42]: Are these changes intended?

File: packages/itwinui-css/scripts/createComponent.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 const fs = require('fs');
-const { copyrightBannerScss, copyrightBannerHtml, copyrightBannerJs } = require('shared/copyrightLinter');
+const { copyrightBannerScss, copyrightBannerHtml, copyrightBannerJs } = require('./copyrightLinter');
```
Comment (rohan-kadkol) [inline packages/itwinui-css/scripts/createComponent.js:6]:
```suggestion
const { copyrightBannerScss, copyrightBannerHtml, copyrightBannerJs } = require('../../../scripts/copyrightLinter');
```

File: scripts/copyrightLinter.js (hunk 1 above)
Comment (mayank99) [reply scripts/copyrightLinter.js:42]: yes they are. i moved the whitespace around to be more readable. the end result is still the same

File: packages/itwinui-css/scripts/createComponent.js (hunk 2 above)
Comment (mayank99) [reply packages/itwinui-css/scripts/createComponent.js:6]: no, the path is correct. they both live in the same folder

Comment (mayank99) [reply packages/itwinui-css/scripts/createComponent.js:6]: ah you're right. we really need to combine these two files 😅
